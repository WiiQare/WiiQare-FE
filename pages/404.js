import Link from "next/link";
import React from "react";
import { BiLeftArrow } from "react-icons/bi";
import Image from "next/image";

const Error404 = () => {
  return (
    <>
      <div className="min-w-screen min-h-screen bg-blue-100 flex items-center p-5 lg:p-20 overflow-hidden relative">
        <div className="flex-1 min-h-full min-w-full rounded-3xl bg-white shadow-xl p-10 lg:p-20 text-gray-800 relative md:flex items-center text-center md:text-left">
          <div className="w-full md:w-1/2">
            <div className="mb-10 lg:mb-20">
              <Image
                alt="logo"
                src="/images/logo_dark.png"
                width={40}
                height={40}
                className="h-10"
              />
            </div>
            <div className="mb-10 md:mb-20 text-gray-600 font-light">
              <h1 className="font-black uppercase text-3xl lg:text-5xl text-orange mb-10">
                You seem to be lost!
              </h1>
              <p>The page you&apos;re looking for isn&apos;t available.</p>
              <p>Try searching again or use the Go Back button below.</p>
            </div>
            <div className="mb-20 md:mb-0">
              <Link href="/" legacyBehavior>
                <a className="text-lg font-light outline-none focus:outline-none transform transition-all hover:scale-110 text-orange hover:text-yellow-600 flex items-center gap-1">
                  <BiLeftArrow /> Go Back
                </a>
              </Link>
            </div>
          </div>
          <div className="w-full md:w-1/2 text-center">
            <Image alt="404" src="/images/404.png" height={40} width={40} />
            <a
              href="https://www.freepik.com/vectors/business"
              target="_blank"
              className="text-xs text-gray-300"
            >
              WiiQare Ã  votre service
            </a>
          </div>
        </div>
        <div className="w-64 md:w-96 h-96 md:h-full bg-blue-200 bg-opacity-30 absolute -top-64 md:-top-96 right-20 md:right-32 rounded-full pointer-events-none -rotate-45 transform"></div>
        <div className="w-96 h-full bg-yellow-200 bg-opacity-20 absolute -bottom-96 right-64 rounded-full pointer-events-none -rotate-45 transform"></div>
      </div>

      <div className="flex items-end justify-end fixed bottom-0 right-0 mb-4 mr-4 z-10">
        <div>
          <a
            title="Buy me a beer"
            href="#"
            target="_blank"
            className="bg-white flex items-center justify-center w-16 h-16 rounded-full transition-all shadow hover:shadow-lg transform hover:scale-110 hover:rotate-12"
          >
            <Image
              alt="favicon"
              className="object-cover object-center w-full h-full rounded-full"
              src="/images/favicon.png"
              width={40}
              height={40}
            />
          </a>
        </div>
      </div>
    </>
  );
};

export default Error404;
