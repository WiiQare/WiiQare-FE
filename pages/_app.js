import Head from "next/head";
import { SessionProvider } from "next-auth/react";
import { ThemeProvider } from "@mui/material/styles";
import { theme } from "../theme";

import "../styles/globals.css";
import "../styles/main.css";

export default function App({ Component, pageProps }) {
  const getLayout = Component.getLayout ?? ((page) => page);
  return (
    <>
      <Head>
        <meta
          name="description"
          content="Payé les soins à vos familles depuis l'étranger"
        />
        <meta name="viewport" content="width=device-width, initial-scale=1" />
        <link rel="icon" href="/favicon.ico" />
        <script src="https://unpkg.com/flowbite@1.5.3/dist/flowbite.js"></script>

      </Head>
      <SessionProvider session={pageProps.session}>
        <ThemeProvider theme={theme}>        
          {getLayout(<Component {...pageProps} />)}
        </ThemeProvider>
      </SessionProvider>
    </>
  );
}
