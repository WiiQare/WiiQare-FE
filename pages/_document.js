import { Html, Head, Main, NextScript } from 'next/document';
import Drawer from '../components/molecules/Navbar/Drawer';

export default function Document() {
  return (
    <Html lang="en">
      <Head>
        <link rel="manifest" href="/manifest.json" />
        <link rel="apple-touch-icon" href="/images/favicon-192x192.png"></link>
        <meta name="theme-color" content="#fff" />
      </Head>
      <script src="https://unpkg.com/flowbite@1.5.3/dist/flowbite.js"></script>
      <body>
        <Main />
        <NextScript />
      </body>
    </Html>
  );
}
