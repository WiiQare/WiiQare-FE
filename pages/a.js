import React, { useEffect } from 'react';
import { useSession } from "next-auth/react";
import Router from "next/router";

const A = () => {
    const {status, data} = useSession();

    useEffect(() => {
       if(status === 'unauthenticated') Router.replace('/login')
    }, [status]);

    if (status === 'authenticated') 
        return (
            <div>
                Protected
            </div>
        )

    return <>Loading...</>
}

export default A;
