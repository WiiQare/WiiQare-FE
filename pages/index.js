import Head from 'next/head'
import { getSession, useSession, signOut } from "next-auth/react"

export default function Home() {

	const { data:session } = useSession()
	
	const handleSignOut = () => {
		signOut()
	}

	return (
		<>
			<Head>
				<title>JAMII - UNICEF CRYPTO</title>
				<meta name="description" content="Payé les soins à vos familles depuis l'étranger" />
				<meta name="viewport" content="width=device-width, initial-scale=1" />
				<link rel="icon" href="/favicon.ico" />
			</Head>
			

			{session ? <User session={session} handleSignOut={handleSignOut} /> : <Guest />}
		</>
	)
}

function User({session, handleSignOut}) {
	return (
		<main>
			Homepage
		</main>
	)
}

function Guest() {
	return (
		<main className='container mx-auto text-center py-20 space-y-8'>
			<h3 className="text-4xl font-bold">Guest User</h3>
		</main>
	)
}

export async function getServerSideProps({req}) {
	const session = await getSession({req})

	if(!session) {
		return {
			redirect: {
				destination: '/login',
				permanent: false
			}
		}
	}

	return {props: {session}}
}