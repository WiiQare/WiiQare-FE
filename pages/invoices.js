import Head from "next/head";
import { getSession } from "next-auth/react";
import DashboardLayout from "../layouts/Dashboard";

const Page = () => {
  return (
    <>
      <Head>
        <title>Invoices</title>
      </Head>
      <div>
        <h2 className="text-black font-w700 mt-4">Invoices</h2>
      </div>
    </>
  );
};

Page.getLayout = (page) => (
  <DashboardLayout className="space-y-8">{page}</DashboardLayout>
);

export default Page;

export async function getServerSideProps({ req }) {
  const session = await getSession({ req });

  if (!session) {
    return {
      redirect: {
        destination: "/login",
        permanent: false,
      },
    };
  }

  return { props: { session } };
}
