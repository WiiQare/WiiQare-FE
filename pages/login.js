import Logo from "../components/atoms/LogoHeader";
import WelcomeSlider from "../components/molecules/WelcomeSlider";
import LoginForm from "../components/organisms/Auth/LoginForm";
import { LOGIN_SLIDES } from "../utils/constants";
import { useMediaQuery, CssBaseline } from "@mui/material";
import Head from "next/head";
import React from "react";

function Login() {
  const matches = useMediaQuery("(max-width: 992px)");
  return (
    <>
      <Head>
        <title>Wiiqare | Login</title>
        <link
          rel="stylesheet"
          href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css"
          integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N"
          crossOrigin="anonymous"
        />
      </Head>
      <CssBaseline />
      <div id="form-section" className="container-fluid signin">
        <Logo />
        <div className="row">
          {!matches && <WelcomeSlider slides={LOGIN_SLIDES} />}
          <LoginForm />
        </div>
      </div>
    </>
  );
}

export default Login;
