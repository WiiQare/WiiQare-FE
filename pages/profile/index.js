import Head from "next/head";
import { getSession } from "next-auth/react";
import DashboardLayout from "../../layouts/Dashboard";
import Profile from "../../components/organisms/Profile";

const Page = () => {
  return (
    <>
      <Head>
        <title>My Profile</title>
      </Head>
        <Profile />
    </>
  );
};

Page.getLayout = (page) => (
  <DashboardLayout className="space-y-8">{page}</DashboardLayout>
);

export default Page;

export async function getServerSideProps({ req }) {
  const session = await getSession({ req });

  if (session) {
    return {
      redirect: {
        destination: "/login",
        permanent: false,
      },
    };
  }

  return { props: { session } };
}
