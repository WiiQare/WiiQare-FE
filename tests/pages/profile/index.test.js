import Page from '@/pages/profile';
import { render } from '@testing-library/react';
import { DrawContext } from '@/pages/_app';
import { SessionProvider } from 'next-auth/react';
require('jest-fetch-mock').enableMocks();

fetch.mockResponse('[]');

describe('Profile', () => {
  it('should render the page', () => {
    const { container } = render(
      <SessionProvider session={{ user: { data: { userId: 'random123' } } }}>
        <DrawContext.Provider
          value={{
            draw: false,
            setDraw: jest.fn(),
            hideSide: false,
            setHideSide: jest.fn(),
          }}
        >
          <Page />
        </DrawContext.Provider>
      </SessionProvider>,
    );
    expect(container).toMatchSnapshot();
  });
});
