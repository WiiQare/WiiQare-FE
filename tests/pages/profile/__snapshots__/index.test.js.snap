// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should render the page 1`] = `
<div>
  <div
    class="p-2 space-y-6 md:py-8 md:px-6 mb-20 md:mb-6"
  >
    <div
      class="border w-full rounded-lg py-8 px-6 flex flex-col flex-wrap md:flex-row justify-between gap-4 bg-white cursor-pointer"
    >
      <div
        class="flex flex-col"
      >
        <h1
          class="md:text-2xl text-gray-700 font-bold flex-wrap flex items-center gap-2 text-lg"
        >
          Mon profil
        </h1>
        <ul
          class="flex gap-2"
        >
          <li>
            <span
              class="flex gap-2 font-semibold text-sky"
            >
              Accueil
              <span>
                /
              </span>
            </span>
          </li>
          <li>
            <span
              class="flex gap-2 font-light text-gray-500"
            />
          </li>
        </ul>
      </div>
      <div
        class="flex flex-row flex-wrap  gap-2 center justify-end space-x-2"
      />
    </div>
    <section
      class="w-full flex flex-col gap-8 items-start pb-20 md:pb-0"
    >
      <div
        class="w-full overflow-hidden md:col-span-2 rounded-lg p-4 flex flex-col gap-6 bg-white drop-shadow-sm"
      >
        <div
          before="+ Cliquez pour changer de couverture"
          class="bg-[url(https://i.goopics.net/46v87b.jpg)] bg-no-repeat relative bg-cover h-56 rounded-lg hover:before:bg-[rgba(0,0,0,.5)] hover:before:cursor-pointer before:transition-all before:duration-200 content before:w-full before:h-full overflow-hidden before:absolute hover:before:content-[attr(before)] before:flex before:justify-center before:items-center before:text-gray-200"
        />
        <div
          class="flex justify-between items-center md:px-6"
        >
          <div
            class="flex gap-5 items-center relative"
          >
            <div
              class="w-20 h-20"
            >
              <img
                alt=""
                class="object-cover rounded-full h-full w-full"
                data-nimg="1"
                decoding="async"
                height="80"
                loading="lazy"
                src="/_next/image?url=https%3A%2F%2Fui-avatars.com%2Fapi%2F%3Fuppercase%3Dtrue%26background%3DFE8023%26name%3Dundefined%26bold%3Dtrue%26color%3DFFF&w=256&q=75"
                srcset="/_next/image?url=https%3A%2F%2Fui-avatars.com%2Fapi%2F%3Fuppercase%3Dtrue%26background%3DFE8023%26name%3Dundefined%26bold%3Dtrue%26color%3DFFF&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fui-avatars.com%2Fapi%2F%3Fuppercase%3Dtrue%26background%3DFE8023%26name%3Dundefined%26bold%3Dtrue%26color%3DFFF&w=256&q=75 2x"
                style="color: transparent;"
                width="80"
              />
            </div>
            <div
              class=""
            >
              <h1
                class="text-sky font-bold text-xl"
              />
              <span
                class="text-xs"
              />
            </div>
          </div>
          <div>
            <button
              class="bg-[rgb(29,170,230,.6)] p-2 border-2 border-sky rounded-md text-white hover:bg-[rgb(29,170,230,.8)] w-fit"
              for="avatar"
            >
              <svg
                fill="currentColor"
                height="22"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="22"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M12 8c-2.168 0-4 1.832-4 4s1.832 4 4 4 4-1.832 4-4-1.832-4-4-4zm0 6c-1.065 0-2-.935-2-2s.935-2 2-2 2 .935 2 2-.935 2-2 2z"
                />
                <path
                  d="M20 5h-2.586l-2.707-2.707A.996.996 0 0 0 14 2h-4a.996.996 0 0 0-.707.293L6.586 5H4c-1.103 0-2 .897-2 2v11c0 1.103.897 2 2 2h16c1.103 0 2-.897 2-2V7c0-1.103-.897-2-2-2zM4 18V7h3c.266 0 .52-.105.707-.293L10.414 4h3.172l2.707 2.707A.996.996 0 0 0 17 7h3l.002 11H4z"
                />
              </svg>
            </button>
            <input
              class="hidden"
              id="avatar"
              name=""
              type="file"
            />
          </div>
        </div>
      </div>
      <div
        class="md:w-3/4 w-full"
      >
        <div
          class="min-full md:col-span-3 rounded-lg p-4 flex flex-col gap-4 bg-white drop-shadow-sm overflow-hidden"
        >
          <div
            class="MuiBox-root css-8atqhb"
          >
            <div
              class="MuiBox-root css-1gsv261"
            >
              <div
                class="MuiTabs-root css-1ujnqem-MuiTabs-root"
              >
                <div
                  class="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
                  style="overflow: hidden; margin-bottom: 0px;"
                >
                  <div
                    aria-label="basic tabs example"
                    class="MuiTabs-flexContainer css-heg063-MuiTabs-flexContainer"
                    role="tablist"
                  >
                    <button
                      aria-controls="simple-tabpanel-0"
                      aria-selected="true"
                      class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                      id="simple-tab-0"
                      role="tab"
                      tabindex="0"
                      type="button"
                    >
                      À propos
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                    <button
                      aria-controls="simple-tabpanel-1"
                      aria-selected="false"
                      class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                      id="simple-tab-1"
                      role="tab"
                      tabindex="-1"
                      type="button"
                    >
                      Paramètres
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                  <span
                    class="MuiTabs-indicator css-1aquho2-MuiTabs-indicator"
                    style="left: 0px; width: 0px;"
                  />
                </div>
              </div>
            </div>
            <div
              class="min-w-full"
            >
              <div
                aria-labelledby="simple-tab-0"
                id="simple-tabpanel-0"
                role="tabpanel"
              >
                <div
                  class="MuiBox-root css-19kzrtu"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                  >
                    <section
                      class="space-y-8 "
                    >
                      <h2
                        class="text-sky font-semibold"
                      >
                        Informations personnelles
                      </h2>
                      <div
                        class="md:w-3/6 space-y-4"
                      >
                        <div
                          class="grid grid-cols-2"
                        >
                          <h5
                            class="font-semibold"
                          >
                            Nom et Prénom :
                          </h5>
                          <span
                            class="text-gray-500"
                          />
                        </div>
                        <div
                          class="grid grid-cols-2"
                        >
                          <h5
                            class="font-semibold"
                          >
                            Email :
                          </h5>
                          <span
                            class="text-gray-500 w-2 md:w-full"
                          />
                        </div>
                        <div
                          class="grid grid-cols-2"
                        >
                          <h5
                            class="font-semibold"
                          >
                            Téléphone :
                          </h5>
                          <span
                            class="text-gray-500"
                          />
                        </div>
                        <div
                          class="grid grid-cols-2"
                        >
                          <h5
                            class="font-semibold"
                          >
                            Age :
                          </h5>
                          <span
                            class="text-gray-500"
                          >
                            --
                          </span>
                        </div>
                        <div
                          class="grid grid-cols-2"
                        >
                          <h5
                            class="font-semibold"
                          >
                            Adresse :
                          </h5>
                          <span
                            class="text-gray-500"
                          >
                            ---
                          </span>
                        </div>
                      </div>
                    </section>
                  </p>
                </div>
              </div>
              <div
                aria-labelledby="simple-tab-1"
                hidden=""
                id="simple-tabpanel-1"
                role="tabpanel"
              />
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
</div>
`;

exports[`Transactions should render the page 1`] = `
<div>
  <div
    class="p-2 space-y-6 md:py-8 md:px-6 mb-20 md:mb-6"
  >
    <div
      class="border w-full rounded-lg py-8 px-6 flex flex-col flex-wrap md:flex-row justify-between gap-4 bg-white cursor-pointer"
    >
      <div
        class="flex flex-col"
      >
        <h1
          class="md:text-2xl text-gray-700 font-bold flex-wrap flex items-center gap-2 text-lg"
        >
          Mon profil
        </h1>
        <ul
          class="flex gap-2"
        >
          <li>
            <span
              class="flex gap-2 font-semibold text-sky"
            >
              Accueil
              <span>
                /
              </span>
            </span>
          </li>
          <li>
            <span
              class="flex gap-2 font-light text-gray-500"
            />
          </li>
        </ul>
      </div>
      <div
        class="flex flex-row flex-wrap  gap-2 center justify-end space-x-2"
      />
    </div>
    <section
      class="w-full flex flex-col gap-8 items-start pb-20 md:pb-0"
    >
      <div
        class="w-full overflow-hidden md:col-span-2 rounded-lg p-4 flex flex-col gap-6 bg-white drop-shadow-sm"
      >
        <div
          before="+ Cliquez pour changer de couverture"
          class="bg-[url(https://i.goopics.net/46v87b.jpg)] bg-no-repeat relative bg-cover h-56 rounded-lg hover:before:bg-[rgba(0,0,0,.5)] hover:before:cursor-pointer before:transition-all before:duration-200 content before:w-full before:h-full overflow-hidden before:absolute hover:before:content-[attr(before)] before:flex before:justify-center before:items-center before:text-gray-200"
        />
        <div
          class="flex justify-between items-center md:px-6"
        >
          <div
            class="flex gap-5 items-center relative"
          >
            <div
              class="w-20 h-20"
            >
              <img
                alt=""
                class="object-cover rounded-full h-full w-full"
                data-nimg="1"
                decoding="async"
                height="80"
                loading="lazy"
                src="/_next/image?url=https%3A%2F%2Fui-avatars.com%2Fapi%2F%3Fuppercase%3Dtrue%26background%3DFE8023%26name%3Dundefined%26bold%3Dtrue%26color%3DFFF&w=256&q=75"
                srcset="/_next/image?url=https%3A%2F%2Fui-avatars.com%2Fapi%2F%3Fuppercase%3Dtrue%26background%3DFE8023%26name%3Dundefined%26bold%3Dtrue%26color%3DFFF&w=96&q=75 1x, /_next/image?url=https%3A%2F%2Fui-avatars.com%2Fapi%2F%3Fuppercase%3Dtrue%26background%3DFE8023%26name%3Dundefined%26bold%3Dtrue%26color%3DFFF&w=256&q=75 2x"
                style="color: transparent;"
                width="80"
              />
            </div>
            <div
              class=""
            >
              <h1
                class="text-sky font-bold text-xl"
              />
              <span
                class="text-xs"
              />
            </div>
          </div>
          <div>
            <button
              class="bg-[rgb(29,170,230,.6)] p-2 border-2 border-sky rounded-md text-white hover:bg-[rgb(29,170,230,.8)] w-fit"
              for="avatar"
            >
              <svg
                fill="currentColor"
                height="22"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="22"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M12 8c-2.168 0-4 1.832-4 4s1.832 4 4 4 4-1.832 4-4-1.832-4-4-4zm0 6c-1.065 0-2-.935-2-2s.935-2 2-2 2 .935 2 2-.935 2-2 2z"
                />
                <path
                  d="M20 5h-2.586l-2.707-2.707A.996.996 0 0 0 14 2h-4a.996.996 0 0 0-.707.293L6.586 5H4c-1.103 0-2 .897-2 2v11c0 1.103.897 2 2 2h16c1.103 0 2-.897 2-2V7c0-1.103-.897-2-2-2zM4 18V7h3c.266 0 .52-.105.707-.293L10.414 4h3.172l2.707 2.707A.996.996 0 0 0 17 7h3l.002 11H4z"
                />
              </svg>
            </button>
            <input
              class="hidden"
              id="avatar"
              name=""
              type="file"
            />
          </div>
        </div>
      </div>
      <div
        class="md:w-3/4 w-full"
      >
        <div
          class="min-full md:col-span-3 rounded-lg p-4 flex flex-col gap-4 bg-white drop-shadow-sm overflow-hidden"
        >
          <div
            class="MuiBox-root css-8atqhb"
          >
            <div
              class="MuiBox-root css-1gsv261"
            >
              <div
                class="MuiTabs-root css-1ujnqem-MuiTabs-root"
              >
                <div
                  class="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
                  style="overflow: hidden; margin-bottom: 0px;"
                >
                  <div
                    aria-label="basic tabs example"
                    class="MuiTabs-flexContainer css-heg063-MuiTabs-flexContainer"
                    role="tablist"
                  >
                    <button
                      aria-controls="simple-tabpanel-0"
                      aria-selected="true"
                      class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                      id="simple-tab-0"
                      role="tab"
                      tabindex="0"
                      type="button"
                    >
                      À propos
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                    <button
                      aria-controls="simple-tabpanel-1"
                      aria-selected="false"
                      class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                      id="simple-tab-1"
                      role="tab"
                      tabindex="-1"
                      type="button"
                    >
                      Paramètres
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                  <span
                    class="MuiTabs-indicator css-1aquho2-MuiTabs-indicator"
                    style="left: 0px; width: 0px;"
                  />
                </div>
              </div>
            </div>
            <div
              class="min-w-full"
            >
              <div
                aria-labelledby="simple-tab-0"
                id="simple-tabpanel-0"
                role="tabpanel"
              >
                <div
                  class="MuiBox-root css-19kzrtu"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                  >
                    <section
                      class="space-y-8 "
                    >
                      <h2
                        class="text-sky font-semibold"
                      >
                        Informations personnelles
                      </h2>
                      <div
                        class="md:w-3/6 space-y-4"
                      >
                        <div
                          class="grid grid-cols-2"
                        >
                          <h5
                            class="font-semibold"
                          >
                            Nom et Prénom :
                          </h5>
                          <span
                            class="text-gray-500"
                          />
                        </div>
                        <div
                          class="grid grid-cols-2"
                        >
                          <h5
                            class="font-semibold"
                          >
                            Email :
                          </h5>
                          <span
                            class="text-gray-500 w-2 md:w-full"
                          />
                        </div>
                        <div
                          class="grid grid-cols-2"
                        >
                          <h5
                            class="font-semibold"
                          >
                            Téléphone :
                          </h5>
                          <span
                            class="text-gray-500"
                          />
                        </div>
                        <div
                          class="grid grid-cols-2"
                        >
                          <h5
                            class="font-semibold"
                          >
                            Age :
                          </h5>
                          <span
                            class="text-gray-500"
                          >
                            --
                          </span>
                        </div>
                        <div
                          class="grid grid-cols-2"
                        >
                          <h5
                            class="font-semibold"
                          >
                            Adresse :
                          </h5>
                          <span
                            class="text-gray-500"
                          >
                            ---
                          </span>
                        </div>
                      </div>
                    </section>
                  </p>
                </div>
              </div>
              <div
                aria-labelledby="simple-tab-1"
                hidden=""
                id="simple-tabpanel-1"
                role="tabpanel"
              />
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
</div>
`;
